camel/toolkits/google_maps_toolkit.py

```diff
@@ -74,57 +74,79 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
     return wrapper
 
 
-class GoogleMapsToolkit(BaseToolkit):
-    r"""A class representing a toolkit for interacting with GoogleMaps API.
+def _import_googlemaps_or_raise() -> Any:
+    r"""Attempts to import the `googlemaps` library and returns it.
 
-    This class provides methods for validating addresses, retrieving elevation,
-    and fetching timezone information using the Google Maps API.
+    Returns:
+        module: The `googlemaps` module if successfully imported.
+
+    Raises:
+        ImportError: If the `googlemaps` library is not installed, this
+        error is raised with a message instructing how to install the
+        library using pip.
     """
+    try:
+        import googlemaps
+
+        return googlemaps
+    except ImportError:
+        raise ImportError(
+            "Please install `googlemaps` first. You can install "
+            "it by running `pip install googlemaps`."
+        )
 
-    def _import_googlemaps_or_raise(self) -> Any:
-        r"""Attempts to import the `googlemaps` library and returns it.
 
-        Returns:
-            module: The `googlemaps` module if successfully imported.
+def _get_googlemaps_api_key() -> str:
```

Comment:
for the checking we can use `@api_keys_required`

Reply:
I think we can use the current implementation because it has extra info about how user can find these API keys. Since Google has a lot of different API keys (for cloud storage, normal apis, etc), I think it would be better to tell user directly where to find these in the error msg. One thing I think would help is adding a `ref` arg to the `@api_keys_required` annotation so that it can give additional info when API keys are not present.